### ISSUE TITLE
no tiled background 

### ISSUE THREAD

**Issue by camillo777** [EvanBacon/expo-rpg#1]
hi, 
I do not get titled background. 
is it the correct behavior? 
Thank you 

Samsung Galaxy Note 4 
Android 6.0.1

**Comment by jgcmarins** [comment]
Same here...
![screenshot_20170610-102200](https://user-images.githubusercontent.com/5133450/27003086-d8fbb50a-4dc6-11e7-884d-e70965fbc36f.png)

Anyway, congratulations for this app. Great job :smile:

**Comment by EvanBacon** [comment]
**FIXED with commit: bd3e482**
Good catch, I need to get an Android device 😅
I'm not sure why this worked on iOS but what happened was, I forgot to setClearColor for the renderer. I couldn't exactly test it but because the background is an `<Image/>` and it sits behind a `<GLView/>` I would imagine that it is fixed now. 
CC @nikki93

**Comment by camillo777** [comment]
@EvanBacon did you already publish to expo repository? is it supposed to update just reloading the expo app again? 
Thank you

**Comment by jgcmarins** [comment]
@camillo777, he has to republish to expo

**Comment by EvanBacon** [comment]
I republished 20 minutes ago 😅, still not working?

**Comment by jgcmarins** [comment]
No :cry:

**Comment by jgcmarins** [comment]
@EvanBacon , it's still without background

**Comment by EvanBacon** [comment]
@jgcmarins I don't have any way of testing it 😱😱 exgl doesn't run on a simulator...

**Comment by jgcmarins** [comment]
No problem. I'll fork it and see if I can help :smile:

**Comment by jgcmarins** [comment]
Hey, I guess you didn't merge the branches...

**Comment by EvanBacon** [comment]
?? I have an empty branch called 'scroll' with no commits to it. Everything else is on master...

**Comment by EvanBacon** [comment]
I figure the easiest solution would be to just use a plane mesh for the background, as opposed to a `<Image />`. I can add, if needed?

**Comment by jgcmarins** [comment]
I don't know why, but pulling `master` branch, leads me to different files.
Take a look at `main.js` here: https://github.com/EvanBacon/expo-rpg/tree/bd3e482c6d4489bb575667281e8531c12ba6e50d
It's different from `main.js` here: https://github.com/EvanBacon/expo-rpg/blob/master/main.js

**Comment by EvanBacon** [comment]
Odd, just pulled and pushed a clean update.

**Comment by jgcmarins** [comment]
Yeah. I've seen. You've refactored it.
Don't worry. My fault.

**Comment by jgcmarins** [comment]
This one: https://threejs.org/docs/index.html#api/math/Plane ?

**Comment by EvanBacon** [comment]
Yeah basically, I saw last night that Expo did something [really similar](https://github.com/expo/floatyplane/blob/master/Game/index.js)
`this.background = Meshes.createBackground(this.width, this.height);
    this.scene.add(this.startScreen);
    this.scene.add(this.planeMesh);
    this.scene.add(this.background);`

**Comment by jgcmarins** [comment]
Nice.
I am new at RN. Same on graphic library. Never tried anything.
I'll study your code, learn three.js and try to help :smile:
